pkg/loader/compose/compose.go

```diff
@@ -693,6 +694,33 @@ func parseEnvironment(composeServiceConfig *types.ServiceConfig, serviceConfig *
 	}
 }
 
+func handleCronJobConcurrencyPolicy(policy string) (batchv1.ConcurrencyPolicy, error) {
+	switch strings.ToLower(policy) {
+	case "allow":
+		return batchv1.AllowConcurrent, nil
+	case "forbid":
+		return batchv1.ForbidConcurrent, nil
+	case "replace":
+		return batchv1.ReplaceConcurrent, nil
+	case "":
+		return "", nil
+	default:
+		return "", fmt.Errorf("invalid cronjob concurrency policy: %s", policy)
+	}
+}
+
+func handleCronJobBackoffLimit(backoffLimit string) (*int32, error) {
+	if backoffLimit == "" {
+		return nil, nil
```

Comment:
I thought about it but I don't want to set a default if kubernetes set the new default to 10 we will need to change it, null will get removed because of the new feature that remove nulls. So this key will be removed if you didn't set it 😅

Reply:
Hmm, yeah I got your point. Actually, Kompose users are mainly Kubernetes beginners, and I believe that having the default attribute of `backoffLimit` already set to default can make users aware of it which is beneficial. This is my POV, leaving the final decision to @cdrage.

Reply:
TBH I agree with @AhmedGrati let's leave it as the Kubernetes default as that'd be best for conversion / learning for the user.